### PR TITLE
Add googleapis to deal with frontend o-auth access codes

### PR DESCRIPTION
**Pull request by thejinxters**

### CODE REVIEW

File: server/controllers/auth_controller.coffee [hunk 1]
```diff
@@ -17,6 +18,46 @@ auth =
...
+          User.methods.findOne { id: google_user.id}, (err, user) ->
+            if err
+              res.status(500).send err
+            else
+              if user
+                user.token = google_user.token
+                response = buildAuthResponse(user)
+                res.status(200).send response
+              else
+                User.methods.addUser google_user, (err, result) ->
```
Comment (Zandrr) [inline server/controllers/auth_controller.coffee:52]: is google_user everything we want?

File: server/models/user.coffee [hunk 2]
```diff
@@ -20,8 +20,22 @@ User.methods =
       if err
         console.log(err)
       if callback
+        console.log "ADD USER RESULT", result
```
Comment (Zandrr) [inline server/models/user.coffee]: is this comment necessary?

**Comment by Zandrr** [comment]
a couple comments - looks good otherwise

File: server/controllers/auth_controller.coffee (hunk 1 above)
Comment (thejinxters) [inline server/controllers/auth_controller.coffee:52]: everything we want to save, at least -- thats google user info, token, and goolgleapis auth info

**Pull request merged**